src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+    action="https://localhost8080"
+    class="form"
+    method="post"
+    data-qa="big">
+
+      <label>
```

Comment:
```suggestion
    <form
      action="https://localhost8080"
      class="form"
      method="post"
      data-qa="big"
    >
      <label>
```
Don't leave new lines between parent and child elements.
Also, please pay attention to your formatting
![image](https://user-images.githubusercontent.com/96409563/202124993-110d991e-21b0-427d-ba94-e876b3f92476.png)